src/Microsoft.Extensions.Caching.Abstractions/MemoryCacheEntryExtensions.cs

```diff
@@ -21,6 +21,20 @@ public static class MemoryCacheEntryExtensions
             return options;
         }
 
+        // TODO: Doc comments
+        public static MemoryCacheEntryOptions SetSize(
+            this MemoryCacheEntryOptions options,
+            long size)
+        {
+            if (size < 0)
```

Comment:
We don't want to impose what size means. When we use this in response caching, it will be bytes but in theory you can have a memory cache where all the sizes are 1 and in that case size means count.